src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -39,10 +39,10 @@ function processAuthoredGametes(authoredChallenge, drakes, state) {
             : state.gametes;
 }
 
-function processAuthoredDrakes(authoredChallenge, trial, template) {
+function processAuthoredDrakes(authoredChallenge, trial, template, unscrambledTrialNumber) {
   // takes authored list of named drakes ("mother", etc) and returns an
   // array specific for this template
-  const authoredDrakesArray = template.authoredDrakesToDrakeArray(authoredChallenge, trial);
+  const authoredDrakesArray = template.authoredDrakesToDrakeArray(authoredChallenge, trial, unscrambledTrialNumber);
```

Comment:
You're right - the variable I added should just be `trialNumber`. It's the *current* variable named `trial` that should change. I renamed it to `authoredTrialNumber` to distinguish it from the `trial` in the state, and show that it is instead an index into the authored trials.